src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectSettingsRemoteController.java

```diff
@@ -64,23 +68,23 @@ public ProjectSettingsRemoteController(MessageSource messageSource, ProjectServi
 			@RequestParam(required = false, defaultValue = "false") boolean forceSync,
 			@RequestParam(required = false, defaultValue = "false") boolean changeUser, Principal principal,
```

Comment:
I would update this to take an object instead of all these individual parameters.

Reply:
This is also an asynchronous response to you should create an ajax controller for this.

Reply:
Updated both in 5bc543eee44a8410455f2cbe94057ef721e15826